services/app-web/src/localAuth/LocalLogin.tsx

```diff
@@ -72,6 +73,13 @@ const localUsers: LocalUserType[] = [
         familyName: 'Coldstart',
         role: 'ADMIN_USER',
     },
+    {
+        id: 'user5',
```

Comment:
We already have a user with the id `user5`
```suggestion
        id: 'user7',
```